### PR TITLE
BCMOHAD-18354 | Updating the web get command

### PR DESCRIPTION
**Pull request by NataliaNikishina**
Please review the PR.

### CODE REVIEW

File: .github/workflows/build-checks.yml [hunk 1]
```diff
@@ -14,7 +14,8 @@ jobs:
 
     - name: 'Install Salesforce CLI'
       run: |
-          wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz
+          
+          wget https://developer.salesforce.com/media/salesforce-cli/sf/channels/stable/sf-linux-x64.tar.xz
           mkdir ~/sfdx
           tar xJf sfdx-linux-x64.tar.xz -C ~/sfdx --strip-components 1
```
Comment (cgijeffolsen) [inline .github/workflows/build-checks.yml:20]: As the filename in line 18 has changed to sf-linux-x64.tar.xz, line 19 has to match.

**Review by cgijeffolsen — CHANGES_REQUESTED** [review]
The Install Salesforce CLI will need the three lines to change together @NataliaNikishina

File: .github/workflows/build-checks.yml (hunk 1 above)
Comment (NataliaNikishina) [reply .github/workflows/build-checks.yml]: @cgijeffolsen Please review the changes. Thank you!

**Review by cgijeffolsen — APPROVED** [review]
Approved @NataliaNikishina thank you.

**Review by RanadheerRG — APPROVED** [review]
Approved @NataliaNikishina

**Pull request merged**